components/gc-subway/subway-doc-en.html

```diff
@@ -0,0 +1,9 @@
+---
+title: GC Subway map
+description: Break up long and complex content into pages that each focus on a step or specific answer people need before moving to the next step or section.
```

Comment:
Let's match the description for FR and EN. Choose which one is more appropriate.

Reply:
Fixed in amended commit.